src/applications/proxy-rewrite/redirects/otherDomainRedirects.json

```diff
@@ -97,7 +97,7 @@
   {
     "domain": "www.benefits.va.gov",
     "src": "/vocrehab/application_roadmap.asp",
-    "dest": "/careers-employment/how-to-apply/"
+    "dest": "/careers-employment/vocational-rehabilitation/how-to-apply/"
```

Comment:
Is this updated in the URL mapping spreadsheet?

Reply:
Yes, it looks like it's been updated.